src/Config/Config.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Bolt\Extension\Animal\Translate\Config;
+
+use Symfony\Component\HttpFoundation\ParameterBag;
+
+/**
+ * General configuration class.
+ *
+ * @author Gawain Lynch <gawain.lynch@gmail.com>
+ */
+class Config extends ParameterBag
+{
+    /**
+     * Constructor.
+     *
+     * @param array $config
+     */
+    public function __construct(array $config)
+    {
+        parent::__construct($config);
+        foreach ($config['locales'] as $name => $parameters) {
+            $this->parameters['locales'][$name] = new Locale($parameters);
+         }
+    }
+
+    /**
+     * @param string $name
+     *
+     * @return Locale
+     */
+    public function getLocale($name)
+    {
+        return $this->get('locales')->get($name);
```

Comment:
`$this->get('locales')` returns an array, right?

So the second `get()` will not work, right?